addons/account/migrations/11.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,201 @@
+account      / account.invoice          / website_message_ids (one2many): NEW relation: mail.message
+account      / account.invoice.line     / is_rounding_line (boolean)    : NEW 
+account      / account.invoice.tax      / amount_rounding (float)       : NEW
+account      / account.journal          / active (boolean)              : NEW 
+account      / account.journal          / bank_statements_source (selection): selection_keys is now '['manual', 'undefined']' ('['manual']')
+account      / account.journal          / color (integer)               : NEW 
+account      / account.journal          / display_on_footer (boolean)   : DEL 
+# NOTHING TO DO
+
+account      / account.move             / statement_line_id (many2one)  : DEL relation: account.bank.statement.line
+account      / account.move.line        / statement_line_id (many2one)  : NEW relation: account.bank.statement.line
+# DONE: post-migration: move statement_line_id values from account.move to account.move.line
+
+account      / account.payment          / message_follower_ids (one2many): NEW relation: mail.followers
+account      / account.payment          / message_ids (one2many)        : NEW relation: mail.message
+account      / account.payment          / message_last_post (datetime)  : NEW 
+account      / account.payment          / state (selection)             : selection_keys is now '['cancelled', 'draft', 'posted', 'reconciled', 'sent']' ('['draft', 'posted', 'reconciled', 'sent']')
+# DONE: set draft payments that have been posted before to cancelled
+account      / account.payment          / website_message_ids (one2many): NEW relation: mail.message
+account      / account.payment          / writeoff_label (char)         : NEW 
+account      / account.payment.term     / sequence (integer)            : NEW required: required, req_default: function
+# DONE: fill sequence field according to original order field 'name'
+account      / account.reconcile.model.template / chart_template_id (many2one)  : NEW relation: account.chart.template, required: required
+account      / account.tax.template     / cash_basis_account (many2one) : NEW relation: account.account.template
+account      / account.tax.template     / tax_exigibility (selection)   : NEW selection_keys: ['on_invoice', 'on_payment']
+# NOTHING TO DO: fields of chart template elements which do not affect installed charts
+account      / res.company              / account_opening_move_id (many2one): NEW relation: account.move
+account      / res.company              / account_setup_bank_data_done (boolean): NEW 
+account      / res.company              / account_setup_bar_closed (boolean): NEW 
+account      / res.company              / account_setup_coa_done (boolean): NEW 
+account      / res.company              / account_setup_company_data_done (boolean): NEW 
+account      / res.company              / account_setup_fy_data_done (boolean): NEW
+# NOTHING TO DO: setup wizards that are harmless or even useful to revisit
```

Comment:
I would set all these fields to True, as we don't want to have these steps as pending on a migrated DB. Someone migrating a DB doesn't need to revisit these configurations steps IMO.

Reply:
Done